hc/accounts/views.py

```diff
@@ -187,6 +188,26 @@ def profile(request):
                                       user=farewell_user).delete()
 
                 messages.info(request, "%s removed from team!" % email)
+        elif "select_allowed_checks" in request.POST:
+            import json
+            post_data = request.POST   # TODO Clean out
+            post_data = json.dumps(dict(post_data))
```

Comment:
`post_data = request.POST.get('key', default)` should suffice here. I don't understand the need for serializing and then deserializing.